src/main/java/net/rptools/maptool/client/functions/getInfoFunction.java

```diff
@@ -358,6 +357,12 @@ private JsonObject getCampaignInfo() throws ParserException {
       state.addProperty("mouseOver", bto.isMouseover() ? BigDecimal.ONE : BigDecimal.ZERO);
       state.addProperty("opacity", bto.getOpacity());
       state.addProperty("order", bto.getOrder());
+      if (bto instanceof FlowColorDotTokenOverlay) {
+        state.addProperty("grid", ((FlowColorDotTokenOverlay) bto).getGrid());
```

Comment:
It would be nice if this key were more explicit, e.g., `"gridSize"` or something like that.

Reply:
Your wish is my command.